code/datums/components/thermite.dm

```diff
@@ -3,7 +3,7 @@
 	var/amount
 	var/overlay
 
-	var/static/list/blacklist = typecacheof(/turf/closed/wall/mineral/diamond)
+	var/static/list/blacklist = typecacheof(/turf/closed/wall/mineral/diamond, /turf/open/lava)
```

Comment:
Can one try to burn through space? Also can this still potentially burn through /turf/closed/indestructible, /turf/open/indestructible, /turf/open/water, /turf/open/space, & /turf/open/chasm?